media-libs/gst-plugins-good/gst-plugins-good-1.20.0.ebuild

```diff
@@ -0,0 +1,54 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+GST_ORG_MODULE="gst-plugins-good"
+
+inherit flag-o-matic gstreamer-meson
+
+DESCRIPTION="Basepack of plugins for GStreamer"
+HOMEPAGE="https://gstreamer.freedesktop.org/"
+
+LICENSE="LGPL-2.1+"
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~mips ~ppc ~ppc64 ~riscv ~sparc ~x86"
+IUSE="+orc"
+
+RDEPEND="
+	>=media-libs/gst-plugins-base-${PV}:${SLOT}[${MULTILIB_USEDEP}]
+	>=app-arch/bzip2-1.0.6-r4[${MULTILIB_USEDEP}]
+	>=sys-libs/zlib-1.2.8-r1[${MULTILIB_USEDEP}]
+	orc? ( >=dev-lang/orc-0.4.17[${MULTILIB_USEDEP}] )
+"
+DEPEND="${RDEPEND}
+	>=dev-util/gtk-doc-am-1.12
```

Comment:
This will be a BDEPEND.

Reply:
I don't think gstreamer uses gtk-doc at all, and therefore a gtk-doc-am dep doesn't make sense. I believe they use hotdoc.

Reply:
Since 1.17.1 gstreamer is disabling gtk_doc feature by default, therefore gtk-doc is not used. Dropped gtk-doc-am dependency for 1.20.0 ebuilds and also for `net-libs/libnice` which has the same issue.